media/com_content/js/admin-articles-default-batch-footer.es6.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+((document, submitForm) => {
+  'use strict';
+
+  // Selectors used by this script
+  const buttonDataSelector = 'data-submit-task';
+  const formId = 'adminForm';
+
+  /**
+   * Submit the task
+   * @param task
+   */
+  const submitTask = (task) => {
+    const form = document.getElementById(formId);
+    if (form && task === 'article.batch') {
+      submitForm(task, form);
+    }
+  };
+
+  // Register events
+  document.addEventListener('DOMContentLoaded', () => {
+    const button = document.getElementById('batch-submit-button-id');
+    if (button) {
+      button.addEventListener('click', (e) => {
+        const task = e.target.getAttribute(buttonDataSelector);
+        submitTask(task);
+      });
+    }
+  });
+})(document, Joomla.submitform);
```

Comment:
sorry, I am late, but this is wrong, 
why do you use `Joomla.submitform` instead of originally used `Joomla.submitbutton`?

`Joomla.submitbutton` made to allow override it by extensions developers, that why we always should use `Joomla.submitbutton` in such cases.

And also, need `(document, Joomla)` instead of `(document, Joomla.submitbutton)`,  in the same reason.

Reply:
Agree, at least with the current code we should use submitButton instead of submitForm. @astridx can you change that?

For `(document, Joomla.submitbutton)`vs `(document, Joomla)` i would prefer to keep it as it is to speedUp lookup and make it easier to test in the future. @Fedik do you see any override problems with that?

Reply:
@dneukirchen I am afraid about that `(document, Joomla.submitbutton)` will capture current method, and will ignore further override. 
But if you think it will work, then can stay `(document, Joomla.submitbutton)`.